### PR TITLE
add new button for editor

### PR DESCRIPTION
**Pull request by chiangkeith**

### CODE REVIEW

File: admin/client/components/VideoSelector.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
+'use strict';
+import { parseVideoAPIResponse, parseImageAPIResponse } from '../../../lib/parseAPIResponse';
+import { Button, Modal, Pagination } from 'elemental';
+import _ from 'lodash';
+import qs from 'qs';
+import xhr from 'xhr';
+import VideoSelection from './VideoSelection';
+import SelectorMixin from './mixins/SelectorMixin';
+import React from 'react';
```
Comment (hcchien) [inline admin/client/components/VideoSelector.js:6]: sorting

File: fields/types/html/editor/atomic-block-processor.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ const processor = {
 				content = Array.isArray(content) ? content : [content];
 				break;
 			case ENTITY.AUDIO.type:
+			case ENTITY.VIDEO.type:
```
Comment (hcchien) [inline fields/types/html/editor/atomic-block-processor.js]: sorting

Comment (chiangkeith) [reply fields/types/html/editor/atomic-block-processor.js]: done, thanks for reminding.

File: admin/client/components/VideoSelector.js (hunk 1 above)
Comment (chiangkeith) [reply admin/client/components/VideoSelector.js:6]: done, thanks for reminding.

**Pull request merged**